test/extended/dr/quorum_restore.go

```diff
@@ -211,15 +206,17 @@ var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Disruptive]", func() {
 					o.Expect(err).NotTo(o.HaveOccurred())
 
 					framework.Logf("Wait for masters to join as nodes and go ready")
-					err = wait.Poll(30*time.Second, 30*time.Minute, func() (done bool, err error) {
+					err = wait.Poll(30*time.Second, 40*time.Minute, func() (done bool, err error) {
 						defer func() {
 							if r := recover(); r != nil {
 								fmt.Println("Recovered from panic", r)
 							}
 						}()
 						nodes, err := oc.AdminKubeClient().CoreV1().Nodes().List(context.Background(), metav1.ListOptions{LabelSelector: "node-role.kubernetes.io/master="})
 						if err != nil {
-							return false, err
+							// scale up to 2nd etcd will make this failure inevitable
+							framework.Logf("return false - err %v", err)
```

Comment:
can you improve this error message to say "ignoring errors while waiting for masters to recover: %v"

Reply:
sure thing

Reply:
hmm rebase is off here let me verify...